packages/web-app-files/tests/unit/mixins/actions/restore.spec.ts

```diff
@@ -128,9 +130,10 @@ function getWrapper({
         }
       },
       $client: {
-        ...sdkMock,
+        ...clientMock,
+        users: { getUser: () => ({ quota: {} }) },
         fileTrash: {
-          ...sdkMock.files,
+          ...clientMock.files,
```

Comment:
shouldn't this be `...clientMock.filesTrash`? It probably works because the APIs are the same, but semantically this does not seem to be correct